cpp/cluster/ClusterProperties.h

```diff
@@ -32,18 +32,19 @@ class ClusterProperties
     ClusterProperties() = default;
 
     //! Compute properties of the point clusters
-    void compute(const freud::locality::NeighborQuery* nq, const unsigned int* cluster_idx);
+    void compute(const freud::locality::NeighborQuery* nq, const unsigned int* cluster_idx,
+                 const float* masses);
```

Comment:
Provide a default value, following: https://github.com/glotzerlab/freud/blob/2075dbdd72686d5ca171fa8d31fa3849fecb1634/cpp/box/Box.h#L377
```suggestion
                 const float* masses = nullptr);
```